OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -0,0 +1,308 @@
+```
+**Note:** Replace variables with values configured in webcenter.env.list
+The above command deletes any previous RCU with the same prefix if **DB_DROP_AND_CREATE=true**
+
+The docker run command creates the container as well as starts the Admin Server in sequence given below:
+
+* Node Manager
+* Admin Server
+
+When the command is run for the first time, we need to create the domain and configure the portal server, so following are done in sequence:
+
+* Loading WebCenter Portal schemas into the database
+* Creating WebCenter Portal domain
+* Configuring Node Manager
+* Starting Node Manager
+* Starting Admin Server
+
+#### B. Stopping Admin Container
+```
+$ docker container stop WCPAdminContainer
+```
+
+#### C. Starting Admin Container
+```
+$ docker container start -i WCPAdminContainer
+```
+
+### 1.3. Portal Container (WCPortalContainer)
+
+#### A. Creating and Running Portal Container
+Run the following command to create the Portal Managed Server container:
+
+```
+$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p <Any Free Port>:$MANAGED_SERVER_PORT -v $DATA_VOLUME:/u01/oracle/user_projects --env-file <directory>/webcenter.env.list $WCPortalImageName /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterPortal.sh; /bin/bash"
+
+```
+**Note:** Replace variables with values configured in webcenter.env.list
+
+The docker run command creates the container as well as starts the WebCenter Portal managed server. 
+
+When the command is run for the first time, WebCenter Content Server connection creation is also done if **CONFIGURE_UCM_CONNECTION=true**.
+ 
+#### B. Stopping Portal Container
+```
+$ docker container stop WCPortalContainer
+```
+
+#### C. Starting Portal Container
+```
+$ docker container start -i WCPortalContainer
+```
+
+#### D. Getting Shell in Portal Container
+```
+$ docker exec -it WCPortalContainer /bin/bash
+```
+
+### 1.4. Elasticsearch Container  (ESContainer)
+To create an Elasticsearch container, we can reuse the environment file `webcenter.env.list` from the above example.
+
+#### A. Data on a volume for Elasticsearch Container
+We need to mount data volume to store crawled data outside the Elasticsearch container.
+
+To mount a host directory `/scratch/wcpdocker/volumes/es` ($ES_DATA_VOLUME) as a data volume, execute the below command.
+
+```
+$ sudo mkdir -p /scratch/wcpdocker/volumes/es
+$ sudo chown 1000:1000 /scratch/wcpdocker/volumes/es
+```
+
+#### B. Creating and Running Elasticsearch Container
+
+```
+$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v $ES_DATA_VOLUME:/u01/esHome/esNode/data --env-file <directory>/webcenter.env.list $WCPortalImageName /bin/bash -c "/u01/oracle/container-scripts/configureOrStartElasticsearch.sh; /bin/bash"
```

Comment:
Same problem here. Please remove the `/bin/bash`.

Reply:
Further, why are you specifying a command on the `run` line anyway? Why doesn't your container image have the correct `ENTRYPOINT` or `CMD` so that this isn't necessary? You can use one of your many environment variables to decide which script to run automatically.